models/dex/dex_info.sql

```diff
@@ -92,6 +92,5 @@ FROM (VALUES
     , ('mento', 'Mento', 'Direct', 'mento')
     , ('mummy_finance', 'Mummy Finance', 'Direct', 'mummyftm')
     , ('carbonhood', 'Carbonhood', 'Direct', 'carbonhood')
-    , ('immortalx', 'ImmortalX', 'Direct', 'immortalx')
```

Comment:
Do you think `ImmortalX` can fit into other market type in this table?

Reply:
There are 3 main market types in this table atm: Direct, Aggregator and Flashloans - I don't think ImmortalX qualifies as any of these.

Reply:
i'd imagine @hildobby added `ImmortalX` -- do you have any opinion on it being in here? or since we're adding it to `perpetual.trades`, maybe it doesn't make sense in `dex.info`?

Reply:
we can add new types ofc, maybe changing it to `Derivatives` or `Perpetual` would make more sense?